database/migration/src/main/resources/changesets/embiggen_metadata_value.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="entry_or_journal_existence_xor">
+        <preConditions>
```

Comment:
The current code appears to work but I think the suggested version would read a lot more intuitively, so I'll try that.